### DISCUSSION TITLE
LearnLaTeX/lesson/01

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#971] [Announcements]
# LearnLaTeX/lesson/01

LaTeX 基础 本课介绍了 LaTeX 是什么以及它将如何工作，并解释了它与 Microsoft Word 或 LibreOffice Writer 等常见文字处理软件的不同之处。 与 Microsoft Word 或 LibreOffice Writer 等常见的文字处理软件不同，LaTeX 通常不提供 WYSIWYG（所见即所得）的功能。使用 L...

https://www.latexstudio.net/LearnLaTeX/lesson/01.html

**Comment by wzangwn** [comment]
稳坐顶楼
